src/instructlab/configuration.py

```diff
@@ -514,12 +514,12 @@ class _train(BaseModel):
     # TODO: could move into its own object.
     # Not strictly necessary for a correct training object.
     phased_phase1_num_epochs: int | None = OPTIONAL_POSITIVE_INTEGER
-    phased_phase1_samples_per_save: int | None = OPTIONAL_POSITIVE_INTEGER
-    phased_phase1_effective_batch_size: int | None = OPTIONAL_POSITIVE_INTEGER
+    phased_phase1_samples_per_save: int | None = -1
```

Comment:
I don't disagree with the approach, but it's not obvious what the difference between `-1`, `0`, and `None` is in this context. We should add a docstring or comment here explaining what they do.

Reply:
yeah good point

Reply:
-1 and 0 have same effect: turns save_samples saving off.
None: use the default `save_samples` value from general training config
>0, not None: overrides general default

Reply:
fixed after slack discussion